monai/transforms/spatial/array.py

```diff
@@ -1064,3 +1065,29 @@ def __call__(self, img, spatial_size=None, mode=None):
             grid[:3] += gaussian(self.rand_offset[None])[0] * self.magnitude
             grid = self.rand_affine_grid(grid=grid)
         return self.resampler(img, grid, mode)
+
+
+class TestIdentity(Transform):
+    """
+    This is a pass through transform to be used for testing purposes.  It allows
+    adding fake behaviors that are useful for testing purposes to simulate
+    how large datasets behave without needing to test on large data sets.
+
+    For example, simulating slow NFS data transfers, or slow network transfers
+    in testing by adding explicit timing delays. Testing of small test data
+    can lead to incomplete understanding of real world issues, and may lead
+    to sub-optimal design choices.
+    """
+
+    def __init__(self, minimum_time_delay=0.0):
```

Comment:
I think we don't have `minumum` and `maxmum` here, so can we change the arg to `delay_time` directly?
I can help to add 1 random transform `RandSimulateDelay` to delay for a randomly picked time from range.
It may be also useful to simulate the real-world dataset with files in different sizes.
Thanks.